lib/oli_web/router.ex

```diff
@@ -723,7 +723,7 @@ defmodule OliWeb.Router do
     live("/:section_slug/progress/:user_id/:resource_id", Progress.StudentResourceView)
     live("/:section_slug/progress/:user_id", Progress.StudentView)
     get("/:section_slug/grades/export", PageDeliveryController, :export_gradebook)
-    get("/:section_slug/updates", PageDeliveryController, :updates)
+    live("/:section_slug/updates", Delivery.ManageUpdates, as: :updates)
```

Comment:
Nit but what do you think about removing the `as:` option here and using the default path name, or instead using a more specific name like `section_updates` or so?

Reply:
We should use the `as` option since we have another route (`/products/:section_slug/updates`) that is served by the same live view, so the route helper will return a bad path for some of those two.

I like to change it to `section_updates`, will go for that one 👍🏼